_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,308 @@
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (center aligned)
+        </li>
+        <li class="item align-center">Blackberries (center aligned)</li>
+    </ul>
+
+    <ul class="list">
+        <li class="item align-right">Apples  (right aligned)</li>
+        <li class="item align-right">Apricots  (right aligned)</li>
+        <li class="item align-right">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (right aligned)
+        </li>
+        <li class="item align-right">Blackberries (right aligned)</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Bulleted List</h3>
+<div class="example-set">
+    <ul class="list list-bulleted">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ul class="list list-bulleted list-circle">
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ul>
+
+    <ul class="list list-bulleted list-disc">
+        <li class="item">Figs</li>
+        <li class="item">Oranges</li>
+        <li class="item">Plums</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Ordered Lists</h3>
+<div class="example-set">
+    <ol class="list list-ordered">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+        <li class="item">Figs</li>
+        <li class="item">Oranges</li>
+        <li class="item">Plums</li>
+    </ol>
+
+    <div class="list list-ordered">
+        <p class="item">Designed on top of an ocean, the town of Darnassea is home to dark elves lead by Colonel Alduin. This town wasn't built by an ocean by accident, as it has ancient, lost technologies, which is of great importance to the people of Darnassea and its success.</p>
+        <p class="item">Darkwell has a troubled economy, which is mainly supported by jewelcrafting, fishing and fletching. But their biggest strengths are intricate jewelcrafting and rare herbalism. However, Darkwell lacks people skilled in farming.</p>
+        <p class="item">Raised inside a geyser field, the port of Darkwell is home to vikings lead by Mrs. Chilson. This port wasn't built by a geyser field by accident, as it has rare resources, which is of great importance to the people of Darkwell and its success.</p>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Link-based Lists</h3>
+<div class="example-set">
+    <nav class="list-links">
+        <span class="item is-current"><a href="http://ux.edx.org">Home</a></span>
```

Comment:
Since you set an "is-current" class on this, I would have a little affordance in the visual display to reflect that state.